src/Compilers/VisualBasic/Portable/BoundTree/Statement.vb

```diff
@@ -1806,6 +1910,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Return OperationKind.None
         End Function
 
+        Protected Overrides ReadOnly Property Children As ImmutableArray(Of IOperation)
```

Comment:
>Protected Overrides ReadOnly Property Children As ImmutableArray(Of IOperation) [](start = 8, length = 79)

BoundUnstructuredExceptionOnErrorSwitch is a lower level node, no need to modify it.